src/main/java/ui/enums/Rank.java

```diff
@@ -0,0 +1,5 @@
+package ui.enums;
+
+public enum Rank {
+    ACE, TWO, THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE, TEN, JACK, QUEEN, KING;
```

Comment:
Just want to make sure, do we want ACE instead of A, KING instead of K, etc because I saw in #29 that Nancy changed the words to characters.

Reply:
I think it should be okay, since these are two different issues that we're tackling when it comes to their usages.